examples/basic_bq/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+   Provider configuration
+  *****************************************/
+provider "google" {
+  version = "~> 1.20.0"
+}
+
+module "bigquery" {
+  source            = "../.."
+  dataset_id        = "${var.dataset_id}"
+  dataset_name      = "${var.dataset_name}"
```

Comment:
I suggest that most of these values be hardcoded rather than provided by required variables. Ideally, every value that can be safely hardcoded while valid in CI should be hardcoded or provided by an other example resource to provide more context about usage.

Reply:
fixed